fs/vfs/fs_dup3.c

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * fs/vfs/fs_dup2.c
+ * fs/vfs/fs_dup3.c
```

Comment:
move to previous line

Reply:
Previous patch ? Yes, obviously.

Reply:
the first patch which add fs_dup3.c but with the wrong comment(fs/vfs/fs_dup2.c)